packages/base/any/kernels/5.10-lts/patches/series.arm64

```diff
@@ -12,7 +12,7 @@
 0016-nvmem-core-allow-to-register-cells-for-existing-device.patch
 0017-nvmem-add-ONIE-NVMEM-cells-support.patch
 0018-userspace-stp.patch
-#0019-pci-EP-reset-fix.patch
```

Comment:
The DentOS commit messages, misses the explanation, why this was commented out in the first place, and why it’s needed. Also the patch in itself seems to have coding style issues. (I only looked at it in the GitHub Web interface.)

Reply:
this patch was disabled during migration to 5.10 kernel. We need it for resolving an issue with resetting PCIe endpoint. w/o it endpoint may not be correctly reconnected